.github/workflows/test.yml

```
@@ -8,12 +8,12 @@ permissions:
 jobs:
   test:
     # os is not in the matrix, because otherwise there would be way too many testing instances
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-24.04
     strategy:
       fail-fast: false
       matrix:
-        node: [14, 16, 18]
-        mongo: [4.2, 5.0]
+        node-version: ['lts/*', 'lts/-1', 'latest']
```

```suggestion
        node: ['lts/*', 'lts/-1', 'latest']
```

Gotta keep the same var name so references to it later still work